index.html

```diff
@@ -287,6 +287,35 @@ <h2>
         and again as an example, the user agent could <a>install</a> the web
         application into a list of bookmarks within the user agent itself.
       </p>
+      <p>
+        A {{Document}} may either be <dfn>installable</dfn> or not. The initial
+        state of a document is non-<a>installable</a>.
+      </p>
+      <p>
+        At any time, the user agent MAY perform the <dfn>steps to determine
+        installability of the document</dfn>:
+      </p>
+      <ol>
+        <li>Let <var>manifest</var> and <var>manifest URL</var> be the result
+        of <a>obtaining the manifest</a>.
+        </li>
+        <li>If <a>obtaining the manifest</a> results in an error, the user
+        agent MAY either:
+          <ol>
+            <li>Fall back to using the <a>top-level browsing context</a>
+            {{Document}}'s metadata to to populate <var>manifest</var> in a
+            user-agent-specific way (e.g., setting
+            <var>manifest</var>.<a data-link-for="WebAppManifest">`name`</a> to
+            the document <a data-cite="HTML#the-title-element">`title`</a>) and
+            considering the document <a>installable</a>, or
```

Comment:
Too easy to miss the "or" right at the end there... 
```suggestion
            considering the document <a>installable</a>.
```